salt/modules/dockerng.py

```diff
@@ -504,10 +504,14 @@ def __virtual__():
                 log.warning(
                     'Insufficient Docker version for dockerng (required: '
                     '{0}, installed: {1})'.format(
-                        '.'.join(docker_versioninfo),
-                        '.'.join(MIN_DOCKER)
-                    )
-                )
+                        '.'.join(map(str, MIN_DOCKER)),
+                        '.'.join(map(str, docker_versioninfo))))
+        else:
+            log.warning(
```

Comment:
Thanks again, @jfindlay.

It should definitely be documented.  From my grepping around previously, I have not found where that is handled.  It is in 2015.8?